tests/resources_in_jar/build.wake.template

```diff
@@ -0,0 +1,15 @@
+global def resourceInJarScalaModule = makeScalaModuleFromJSON here "resources_in_jar"
+
+def test Unit =
+  def paths = scalaModuleClasspath resourceInJarScalaModule
+  def classpath = paths | map getPathName | catWith ":"
+  def cmd = which "java", "-cp", classpath, "test.Main", Nil
+  def visible = paths
+  def job = makePlan cmd visible | runJob
+  match job.getJobStatus
+    Exited 0 = Pass "Passed"
+    Exited i = Fail "Exited with status {i.format}".makeError
+    Signaled i = Fail "Signaled with status {i.format}".makeError
+    Aborted e = Fail e
+
+publish apiScalaSiFiveTests = test, Nil
```

Comment:
FYI, I'm not sure if this pattern will scale. When a test fails, you're going to want to be able to run an individual test instead of all them, but there's no way to select an individual test. I would have recommended defining a `Test` tuple or something similar that must have a unique `name` property, that at least allows you to invoke the test independently.

Reply:
Yeah fair point, I agree that a `Test` tuple or something is better, will do in follow on stuff